app/assets/javascripts/application.js

```diff
@@ -1,4 +1,7 @@
 //= require jquery/dist/jquery.min.js
 //= require click-tracker.js
+//= require media-player.js
+//= require govuk_elements/public/javascripts/vendor/details.polyfill.js
```

Comment:
Maybe we should add `vendor/assets/bower_components/govuk_elements/public/javascripts/vendor/` to our asset path then we can just require `details.polyfill.js`?